resources/lua/naws.lua

```diff
@@ -0,0 +1,39 @@
+-- See https://www.rfc-editor.org/rfc/rfc1073.html
+local NAWS_PROTOCOL = 31
+local naws_enabled = false
+
+-- return table with the network byte order (e.g. big endian) encoding of the
+-- width and height given.
+local function network_dimensions(width, height)
+    bytes = {}
+    local append_byte = function(c)
+      table.insert(bytes, string.byte(c))
+    end
+    string.pack(">i2", width):gsub(".", append_byte)
+    string.pack(">i2", height):gsub(".", append_byte)
+    return bytes
```

Comment:
Is the big endian manipulation needed? I was under the impression that this is handled in the lower network layers. However if you've tested and it works then it works.

On another note, does NAWS expect the entire terminal height or the height of the "output area". In blightmud this will be `height - 2 (for input area and prompt area) - status_area_height`.

Reply:
> Is the big endian manipulation needed? I was under the impression that this is handled in the lower network layers. However if you've tested and it works then it works.

I'll do some experimentation to see if I can remove it outright. I can confirm I did test the data transmitted with this version matches what I'd expect to see server-side for a given width/height client-side :+1: 

> On another note, does NAWS expect the entire terminal height or the height of the "output area"

Hmmm, great point. I think it's most useful if we send the height of the output area since the server wants to know the boundaries of output it can draw and that won't include the input/prompt/status areas. I'll adjust.

Reply:
> I'll do some experimentation to see if I can remove it outright

It might be my own Lua inexperience but I can't find a way to create the right byte table encoding of the two ints to transmit without packing them using the stdlib string method or manually with bitwise operations.

>  I'll adjust.

I've updated the terminal height reported via NAWS to account for the input/status areas. PTAL: https://github.com/Blightmud/Blightmud/pull/712/commits/5fdd2f76b8b93f6467e50c1bc7f52b7d3e84154b